tools/extras/install_kaldi_io.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Install kaldi_io. Please refer to https://github.com/vesis84/kaldi-io-for-python 
+# for details.
+
+python3 -m pip install --user kaldi_io
```

Comment:
Can somehow explain to me how this invocation makes kaldi_io available?  Does this mean that Karel has somehow registered that module with some official repo?

Reply:
I don't know how it works, but it seems like Karel must've done something like that. If you copy and paste python3 -m pip install --user kaldi_io to the terminal it finds the package and starts installing it.